tests/unit/test_eq.py

```diff
@@ -97,11 +170,41 @@ def test_create_eq_constructor_missing_uprn(self):
                 self.uac_json, None, self.app, None).build()
         self.assertIn('Attributes is empty', ex.exception.message)
 
-    def test_build_display_address(self):
+    def test_build_display_address_en(self):
         from app import eq
```

Comment:
Is this import required, or could the code be written as `app.eq.EqPayloadConstructor...`?

Reply:
Left as is - function before has following comment where import is also made, so not sure of the effect of changing
'# NB: local import to avoid overwriting the patched version for some tests'